### ISSUE TITLE
Can stereographic logits be rewritten using `geoopt.manifolds.stereographic.math.dist2plane`?

### ISSUE THREAD

**Issue by pchlenski** [pchlenski/manify#20]
Per the kappa-GCN paper ([link](https://proceedings.mlr.press/v119/bachmann20a/bachmann20a.pdf)), logits in non-Euclidean models are conventionally computed as distances to a hyperplane. 

Currently, the logits implementation relies on some fairly complex logic. Can it be rewritten more simply using dist2plane?
